frontend/src/components/feature/chat/ChatMessage/MessageActions/MessageActions.tsx

```diff
@@ -97,9 +108,16 @@ export const MessageContextMenu = ({ message, onDelete, onEdit, onReply, onForwa
                 </ContextMenu.Item> */}
 
                 </ContextMenu.Group>
-
-
-
+                
+                {Object.keys(reactions).length !== 0 && <ContextMenu.Group>
+                    <ContextMenu.Separator />
+                    <ContextMenu.Item onClick={openReactionAnalytics}>
+                        <Flex gap='2'>
+                            <MdOutlineEmojiEmotions size='18' />
+                            Reaction Analytics
```

Comment:
Can we rename this to View Reactions instead?
```suggestion
                            View Reactions
```